packages/python3.xlmmacrodeobfuscator.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,8 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = "XLMMacroDeobfuscator"
+$category = 'Office'
+
+VM-Execute-Custom-Command -name "Uninstall with python3 pip" -cmds "pip uninstall $toolName -y"
+VM-Uninstall $toolName $category
```

Comment:
`VM-Uninstall` is undoing more things that what you are doing in the installer (it calls `Uninstall-BinFile`). I would copy only the lines in `VM-Uninstall` that apply here.

Reply:
`VM-Remove-Tool-Shortcut` is more fitting.